tests/view_tests/tests/test_i18n.py

```diff
@@ -37,6 +37,12 @@ def test_setlang(self):
         response = self.client.post('/i18n/setlang/', post_data, HTTP_REFERER='/i_should_not_be_used/')
         self.assertRedirects(response, '/')
         self.assertEqual(self.client.session[LANGUAGE_SESSION_KEY], lang_code)
+        # The code also sets the language in a cookie (see #15902 for the plan)
```

Comment:
I would write this comment so it doens't have to be rewritten later: `# The language is set in a cookie.`